src/pages/create/components/stepGetter.ts

```
@@ -50,6 +44,14 @@ const stepGetter = (stepParam: string | undefined): Step | never => {
         stepNameTranslation: t("createGroup.complete.stepName"),
       };
       break;
+    case "name":
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)